test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -1422,8 +1422,12 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 				Expect(err).NotTo(HaveOccurred())
 
 				By("Ensuring we can create a curl pod to connect to the service")
-				ilbCurlPod, err := pod.RunLinuxWithRetry("byrnedo/alpine-curl", "curl-to-ilb", "default", fmt.Sprintf("curl %s", sILB.Status.LoadBalancer.Ingress[0]["ip"]), false, 1*time.Minute, cfg.Timeout)
-				Expect(err).NotTo(HaveOccurred())
+				for i := 0; i < 100; i++ {
+					ilbCurlPod, err := pod.RunLinuxWithRetry("byrnedo/alpine-curl", "curl-to-ilb", "default", fmt.Sprintf("curl %s", sILB.Status.LoadBalancer.Ingress[0]["ip"]), false, 1*time.Minute, 1*time.Second, 30*time.Second)
```

Comment:
this test (which we run 100 times in succession) will fail if it takes longer than one second to curl to the LB IP